src/v/cluster/controller.cc

```diff
@@ -304,11 +304,59 @@ ss::future<> controller::start(cluster_discovery& discovery) {
           return _stm.invoke_on(controller_stm_shard, &controller_stm::start);
       })
       .then([this] {
-          return _stm.invoke_on(controller_stm_shard, [](controller_stm& stm) {
-              // we do not have to use timeout in here as all the batches to
-              // apply have to be accesssible
-              return stm.wait(stm.bootstrap_last_applied(), model::no_timeout);
-          });
+          auto disk_commit_offset = _raft0->log().offsets().committed_offset;
```

Comment:
we can use `dirty_offset` here, we once had plans to remove the `committed_offset` completely. When staring both of the are equal.

Reply:
done